packages/data-stores/src/launch/selectors.ts

```diff
@@ -27,20 +27,6 @@ export const getSelectedDomain = ( state: State ): DomainSuggestions.DomainSugge
 	state.domain;
 export const getSelectedPlanProductId = ( state: State ): number | undefined => state.planProductId;
 
-/**
- * Returns the product id of the the last paid plan the user had picked.
- * If they revert to a free plan,
- * this is useful if you want to recommend their once-picked paid plan
- *
- * @param state State
- */
-export const getPaidPlanProductId = ( state: State ): number | undefined => {
-	const productId = state.planProductId;
-	const isFree = select( PLANS_STORE ).isPlanProductFree( productId );
-
-	return productId && ! isFree ? state.planProductId : undefined;
-};
-
```

Comment:
I agree we don't need this selector as it is but if you look at changes from https://github.com/Automattic/wp-calypso/pull/48452 we are using twice this selector but just to check if the selected plan is a paid plan so `isSelectedPlanPaid` might be a good candidate for a selector on Launch store.